carberretta/utils/chron.py

```diff
@@ -1,5 +1,6 @@
 import datetime as dt
 from time import strftime
+from json import JSONEncoder
```

Comment:
Alphabetise imports as per PEP 8 standards.
```suggestion
from json import JSONEncoder
from time import strftime
```